buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -593,6 +719,34 @@ public void testSequentialShortAccess() {
     }
 
     @Test
+    public void testSequentialShortLEAccess() {
+        buffer.writerIndex(0);
+        for (int i = 0; i < buffer.capacity(); i += 2) {
+            short value = (short) random.nextInt();
+            assertEquals(i, buffer.writerIndex());
+            assertTrue(buffer.isWritable());
+            buffer.writeShortLE(value);
```

Comment:
consider sharing code with the previous test. This line looks like the only difference?  Maybe just have a method:

```java
private static void testSequentialShortAccees(boolean testBigEndian) {
..
}
```